src/fidesops/service/connectors/query_config.py

```diff
@@ -320,9 +324,6 @@ def generate_query(
                         self.format_clause_for_query(string_path, "IN", string_path)
                     )
                     query_data[string_path] = tuple(data)
-                else:
-                    #  if there's no data, create no clause
-                    pass
```

Comment:
Removed as it's not necessary.

Reply:
This is just a Python cleanup, if you are specifying if/elif/elif, and your "else" case is "do nothing", you don't need it. 

```python3
def animal_sounds(animal:str):
    if animal == "dog":
        print("woof")
    elif animal == "cat":
        print("meow")
```

```bash
>>> animal_sounds("dog")
woof
>>> animal_sounds("cat")
meow
>>> animal_sounds("toad")
```

Thanks for pointing out this is elsewhere, I'll clean it up.

Reply:
oh right, good point 😁